main/src/ca/uwaterloo/flix/api/lsp/provider/SemanticTokensProvider.scala

```diff
@@ -230,7 +230,7 @@ object SemanticTokensProvider {
     * Returns all semantic tokens in the given signature `sig0`.
     */
   private def visitSig(sig0: TypedAst.Sig): Iterator[SemanticToken] = sig0 match {
-    case TypedAst.Sig(sym, spec, exp) =>
+    case TypedAst.Sig(sym, spec, exp, loc) =>
```

Comment:
This probably also unused.

Reply:
@LoZander If you use intellij idea it should warn you about unused variables.

Reply:
> @LoZander If you use intellij idea it should warn you about unused variables.

But don't bother fixing all unused variables that you did not introduce yourself. That's out of scope for this PR.

Reply:
Yes, just focus on your code.

Reply:
Yes, these ones slipped by me. I'll fix it :)